app/src/components/pages/divisionPage/divisionPage.tsx

```diff
@@ -49,8 +47,12 @@ export default function DivisionPage({ data }: any) {
     params.set("page", pageNumber.toString());
     setCurrentPage(pageNumber);
     const url = `${pathname}?${params.toString()}#${data.slug}`;
-    replace(url);
-    headingRef.current?.focus;
+    setIsLoaded(false);
+    push(url);
```

Comment:
just curious but did using `push` in place of `replace` make a difference?

Reply:
Not that i noticed? But Edwin mentioned this specifically so I thought might as well